slack_sdk/models/blocks/block_elements.py

```diff
@@ -326,19 +362,28 @@ def __init__(
         self,
         *,
         action_id: Optional[str] = None,
-        placeholder: Optional[str] = None,
```

Comment:
Button block element does not accept placeholder. This has been wrong for a long time.